packages/web/components/bridge/bridge-provider-dropdown.tsx

```diff
@@ -114,12 +145,10 @@ export const BridgeProviderDropdown = ({
                   expectedOutputFiat,
                 },
               }) => {
-                const totalFee = transferFeeFiat
-                  ?.add(
-                    gasCostFiat ??
-                      new PricePretty(transferFeeFiat.fiatCurrency, 0)
-                  )
-                  .toString();
+                const totalFee = getTotalFee({
+                  transferFeeFiat: transferFeeFiat,
+                  gasCostFiat: gasCostFiat,
+                })?.toString();
                 const isSelected = selectedQuote.provider.id === provider.id;
```

Comment:
This to me is a code smell that it'd be ideal to do the totaling where quotes are constructed since it's purely a function of quote data, vs here. No required change just wanted to make a remark.

Reply:
That's a good idea! Should be a simple change. I'll push a fix